dotCMS/src/main/java/com/dotcms/graphql/business/PageAPIGraphQLTypesProvider.java

```diff
@@ -237,7 +246,64 @@
         typeMap.put("Geolocation", TypeUtil.createObjectType("Geolocation",
                 geoFields));
 
+        final Map<String, TypeFetcher> layoutFields = new HashMap<>();
+        layoutFields.put("width", new TypeFetcher(GraphQLLong,
+                new PropertyDataFetcher<TemplateLayout>("width")));
+        layoutFields.put("title", new TypeFetcher(GraphQLString,
+                new PropertyDataFetcher<TemplateLayout>("title")));
+        layoutFields.put("header", new TypeFetcher(GraphQLString,
+                new PropertyDataFetcher<TemplateLayout>("header")));
+        layoutFields.put("footer", new TypeFetcher(GraphQLString,
+                new PropertyDataFetcher<TemplateLayout>("footer")));
+        layoutFields.put("body", new TypeFetcher(GraphQLTypeReference.typeRef("Body"),
+                new PropertyDataFetcher<TemplateLayout>("body")));
+
+        typeMap.put("Layout", TypeUtil.createObjectType("Layout",
+                layoutFields));
+
+        final Map<String, TypeFetcher> bodyFields = new HashMap<>();
+        bodyFields.put("rows", new TypeFetcher(list(GraphQLTypeReference.typeRef("LayoutRow")),
+                new PropertyDataFetcher<TemplateLayout>("rows")));
+
+        typeMap.put("Body", TypeUtil.createObjectType("Body",
+                bodyFields));
+
+        final Map<String, TypeFetcher> rowFields = new HashMap<>();
+        rowFields.put("columns", new TypeFetcher(list(GraphQLTypeReference.typeRef("LayoutColumn")),
+                new PropertyDataFetcher<TemplateLayoutRow>("columns")));
+        rowFields.put("styleClass", new TypeFetcher((GraphQLString),
```

Comment:
Codacy found an issue: [The String literal "styleClass" appears 4 times in this file; the first occurrence is on line 274](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6084048)